packages/modular-scripts/src/addPackage.ts

```diff
@@ -50,6 +50,7 @@ async function addPackage({
         choices: [
           { title: 'A plain package', value: 'package' },
           { title: 'A view within an application', value: 'view' },
```

Comment:
Maybe to be clear what the difference between view and esm-view, could we update the title of view to indicate it's not an esm?

Reply:
@cangarugula Happy to change it. Not sure of how to word this though, "A non-ESM view within an application" sounds bad. Do you have suggestions?

Reply:
I'll bring some more options to the table to help us think through what we're trying to convey.

The first and most concise is to simply say what these things are and require the reader to view the docs to understand what "view" and "app" mean.

- A plain package
- A view compiled to ES Modules
- A view compiled to a bundle file
- A standalone app
- Choose my own template

Following my previous logic, we could even shorten "A standalone app" to "An app".

However, if we wanted to be more descriptive, we could throw in an indication of how these things relate to micro-frontends, which is modular's _thing_:

- A plain package
- A micro-frontend view compiled to ES Modules
- A micro-frontend view compiled to a bundle file
- A micro-frontend container app
- Choose my own template

We can even shorten things a little bit:

- A plain package
- A micro-frontend view (ES Modules)
- A micro-frontend view (bundle file)
- A micro-frontend container app
- Choose my own template

What do people think?

Reply:
Should we add the `type` to better disambiguate?

- A plain package (package)
- A micro-frontend view compiled to ES Modules (esm-view)
- A micro-frontend view compiled to a bundle file (view)
- A micro-frontend container app (app)
- Choose my own template

@cangarugula @steveukx

Reply:
@cangarugula and @benpryke chose Option C. I incorporated it and:

1) Incorporated @benpryke's suggestion of not repeating the word "app"
2) Incorporated @cangarugula's preference of specifying "react" for esm-views, views and apps